microk8s-resources/default-hooks/remove.d/10-cni-link

```diff
@@ -4,11 +4,15 @@
 
 if ! is_strict || (is_strict && snapctl is-connected network-control)
 then
-  for link in cni0 calico.vxlan
+  for link in cni0
   do
     if "${SNAP}/sbin/ip" link show "${link}"
     then
       "${SNAP}/sbin/ip" link delete "${link}" || true
     fi
   done
+  for calink in $("${SNAP}/sbin/ip" -j link show |\
+                  "${SNAP}/usr/bin/jq" -r '.[].ifname | select(test("^vxlan.calico|cali[a-f0-9]*$"))')
+  do "${SNAP}/sbin/ip" link delete "${calink}"
```

Comment:
Please add a `|| true` at the end of this line so that if for any reason the deletion fail we do not fail the entire removal of the snap.

Reply:
Ah, you're right, thanks a lot! I've added the `|| true` clause.